frontend/packages/topology/src/components/workload/resource-alert.tsx

```diff
@@ -77,3 +80,55 @@ export const useHealthChecksAlert = (element: GraphElement): DetailsResourceAler
       }
     : null;
 };
+
+export const useResourceQuotaAlert = (element: GraphElement): DetailsResourceAlertContent => {
+  const { t } = useTranslation();
+  const resource = getResource(element);
+  const name = resource?.metadata?.name;
+  const namespace = resource?.metadata?.namespace;
+
+  const canUseAlertAction = useAccessReview({
+    group: DeploymentModel?.apiGroup,
+    resource: DeploymentModel?.plural,
```

Comment:
DeploymentModal is a const, we don't need an optional check here:

```suggestion
    group: DeploymentModel.apiGroup,
    resource: DeploymentModel.plural,
```